app/components/site-chrome/template.hbs

```diff
@@ -148,6 +148,9 @@
     <!-- .x-scrollable-body -->
 
     {{#if media.isLargeAndUp}}
+    <div class="sitechrome__nav-footer">
```

Comment:
didn't want to rename the rest of the sitechrome classes yet. Was mostly focused on the new component. Using `site-chrome` makes sense though.